packages/compass/src/main/import-export-connections.ts

```diff
@@ -0,0 +1,18 @@
+import { promises as fs } from 'fs';
+import type { ExportConnectionOptions, ImportConnectionOptions } from 'mongodb-data-service';
+
+export async function doExportConnections(filename: string, options: ExportConnectionOptions = {}): Promise<void> {
+  console.log(`Exporting connections to "${filename}" (${options.passphrase ? 'with' : 'without'} passphrase)`);
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const { exportConnections } = await import('mongodb-data-service');
```

Comment:
Question, is there a reason why we dynamically import the data service here and below?

Reply:
Not a good one probably :) So far we’re always importing data-service in the renderer, I didn’t want to add it and its dependencies to the main process startup sequence since 99 % of the time it’s not going to be used. Can also move this to a regular import if you prefer.

Reply:
Sounds good, I like the optimization, can we leave that comment you've just put as a comment there to help some future readers?

Reply:
Sure, done! :+1: